tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -219,21 +215,41 @@ def delete_test_user(email: str):
 
 
 def _get_user_details_via_api(user_email: str):
-    users = admin_client.get("/api/user-management/users").json()
+    response = admin_client.get("/api/user-management/users")
+
+    if response.status_code == 404:
```

Comment:
I don't think we should be expecting a 404 here - just an empty list.

Reply:
See other comment